pages/linux/hwinfo.md

```diff
@@ -0,0 +1,24 @@
+# hwinfo
+
+> Probe for the hardware present in the system.
+> More information: <https://github.com/openSUSE/hwinfo>.
+
+- Get graphics card information:
+
+`hwinfo --gfxcard`
+
+- Get network device information:
+
+`hwinfo --network`
+
+- See other type of hardware items:
+
+`hwinfo --help`
+
+- Abbreviate the output:
+
+`hwinfo --short --disk --cdrom`
+
+- Write all hardware info to a file:
+
+`hwinfo --all --log={{path/to/file}}`
```

Comment:
We usually put the help options at the end of a page and give it a simple "Display help description. At when I've commited it's been corrected that way. 
```suggestion
- Abbreviate the output:

`hwinfo --short --disk --cdrom`

- Write all hardware info to a file:

`hwinfo --all --log={{path/to/file}}`

- Display help:

`hwinfo --help`
```